templates/items/details/details-spell.hbs

```diff
@@ -0,0 +1,232 @@
+                         ariaLabel=(localize "DND5E.Prepared") }}
+            {{/if}}
+
+            {{!-- Mode --}}
+            {{ formInput fields.preparation.fields.mode value=source.preparation.mode
+                         choices=config.spellPreparationModes labelAttr="label" }}
+        </div>
+    </div>
+
+    {{!-- Source Class --}}
+    {{#if isEmbedded}}
+    {{ formField fields.sourceClass value=source.sourceClass localize=true choices=item.parent.spellcastingClasses
+                 labelAttr="name" blank="" }}
+    {{/if}}
+</fieldset>
+
+<fieldset>
+    <legend>{{ localize "DND5E.Casting" }}</legend>
+
+    {{!-- Activation --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.SpellCastTime" }}</label>
+        <div class="form-fields">
+
+            {{!-- Amount --}}
+            {{#if (and source.activation.type (not isCostlessAction))}}
+                {{ formField fields.activation.fields.value value=source.activation.value label="DND5E.Amount" placeholder="—"
+                             localize=true hint=false classes="label-top" }}
+            {{/if}}
+
+            {{!-- Type --}}
+            {{ formField fields.activation.fields.type value=source.activation.type label="DND5E.Cost" localize=true
+                         classes="label-top" hint=false choices=config.abilityActivationTypes blank="" }}
+        </div>
+
+        {{!-- Condition --}}
+        {{ formInput fields.activation.fields.condition value=source.activation.condition input=inputs.createTextInput
+                     placeholder=(localize "DND5E.ItemActivationCondition") classes="full-width" }}
+    </div>
+
+    {{!-- Range --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.Range" }}</label>
+        <div class="form-fields">
+
+            {{!-- Value --}}
+            {{#if system.hasScalarRange}}
+            {{ formField fields.range.fields.value value=source.range.value label="DND5E.Value" localize=true
+                         hint=false classes="label-top" }}
+            {{/if}}
+
+            {{!-- Units --}}
+            {{ formField fields.range.fields.units value=source.range.units label="DND5E.MovementUnits" localize=true
+                         hint=false blank="DND5E.None" localize=true options=rangeTypes classes="label-top" }}
+        </div>
+
+        {{!-- Condition --}}
+        {{#if source.range.units}}
+        {{ formInput fields.range.fields.special value=source.range.special input=inputs.createTextInput
+                     placeholder=(localize "DND5E.RANGE.FIELDS.range.special.label") localize=true
+                     classes="full-width" }}
+        {{/if}}
+    </div>
+
+    {{!-- Duration --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.Duration" }}</label>
+        <div class="form-fields">
+
+            {{!-- Amount --}}
+            {{#if system.hasScalarDuration}}
```

Comment:
(there is a bug in the `.scalar` values on these fields at the moment that I fixed in my other PR)

```suggestion
            {{#if system.duration.scalar}}
```